### DISCUSSION TITLE
How does tabby draw its window borders in KDE Plasma session? I don't see the custom window border decoration applied to tabby.

### DISCUSSION THREAD

**Discussion by denppa** [Eugeny/tabby#8810] [General]
# System Info
Debian Bookworm
KDE Wayland Session
Tabby installed via `apt` from deb repo
Tabby seems to be running under xwayland from the output of `clsclients`

# Question
The `bismuth` plugin of KDE plasma desktop environments supplies a custom border decoration in the style of a tiling window manager, it applied to most every app I use on the system, whether installed via flatpak or deb. Yet tabby alone doesn't seem to use the same method of drawing borders, and thus the Bismuth decoration does not apply to tabby.

Is there a way to get the tiling borders for tabby or would this be impossible?

Also, what framework is tabby built upon? QT, GTK, or perhaps electron?

**Comment by Eugeny** [comment]
You probably need to switch window frame to "native" in Settings -> Window. Tabby uses Electron and a custom window border by default.

**Reply by denppa** [reply]
This is the solution. Head into settings and set tabby to use system window decorations, then it will have a window border drawn by the system after a restart. Thanks.
